app/static/js/initialize_map.js

```diff
@@ -0,0 +1,155 @@
+geocoder = new google.maps.Geocoder();
+address = document.getElementById("address");
+//Initial map center coordinates
+INITIAL_CENTER_LAT = 39.952;
+INITIAL_CENTER_LONG = -75.195;
+initial_coords = new google.maps.LatLng(INITIAL_CENTER_LAT, INITIAL_CENTER_LONG);
+//Initial map zoom value
+ZOOM = 17;
```

Comment:
What should we do rather than hardcoding zoom? Note that the user still has the ability to adjust the zoom value from the ui.